src/fzx/tui/term_app.cpp

```diff
@@ -147,24 +148,32 @@ void TermApp::redraw()
           ++p;
           if (!highlighted) {
             highlighted = true;
-            mTTY.put("\x1B[33m"sv);
+            mTTY.setFg(mPalette.mMatchFg);
           }
         } else {
           if (highlighted) {
             highlighted = false;
-            mTTY.put("\x1B[0m"sv);
+            if (iscursor) {
+              mTTY.setFg(mPalette.mCursorFg);
+            } else {
+              mTTY.setFg(mPalette.mDefaultFg);
```

Comment:
defaultFg still doesn't really work here, it's only ever set here. defaultFg has to be also set before the loop and after `mTTY.clearColor()`
![screenshot-231226-095517](https://github.com/ii14/fzx/assets/59243201/f21a8b7a-de60-4878-a0f2-d63aeba8964b)

Reply:
done